src/util/logger-utils.ts

```diff
@@ -4,7 +4,7 @@ import bformat from "bunyan-format";
 import { DEBUG } from "bunyan";
 import { createHashWithSharedSecret } from "utils/encryption";
 
-const SENSITIVE_DATA_FIELDS = ["jiraHost", "orgName", "repoName", "userGroup", "userGroup", "aaid", "username"];
+const SENSITIVE_DATA_FIELDS = ["jiraHost", "orgName", "repoName", "userGroup", "userGroup", "aaid", "username", "prTitle", "prRef"];
```

Comment:
As we find more UGC/PII we continue to add to this list.
Its manageable at the moment but may need to consider moving it to a config file of some sort if it gets too massive!!